### PR TITLE
Feature/register form

### PR DESCRIPTION
**Pull request by joejemmely**

### CODE REVIEW

File: pages/register.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+import superagent from 'superagent';
+
+import { businessDays } from '../utils/businessDays';
+import { businessHours } from '../utils/businessHours';
+import { instruments } from '../utils/instruments';
+
+export default function RegisterPage() {
+  async function handleSubmit(values) {
+    console.log(values);
+    // const response = await superagent.post(
```
Comment (vladimirsan) [inline pages/register.js]: no commented out code

File: pages/register.js [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+import { Formik, Field, FieldArray, Form, ErrorMessage } from 'formik';
+import superagent from 'superagent';
+
+import { businessDays } from '../utils/businessDays';
+import { businessHours } from '../utils/businessHours';
+import { instruments } from '../utils/instruments';
+
+export default function RegisterPage() {
+  async function handleSubmit(values) {
+    console.log(values);
```
Comment (vladimirsan) [inline pages/register.js]: console.log is ok, but let's keep it away from master.

Comment (vladimirsan) [reply pages/register.js]: or let's have a system where it doesn't deploy the console.log

File: pages/register.js [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    // const response = await superagent.post(
+    //   'http://localhost:3000/api/register',
+    //   JSON.stringify(values)
+    // );
+    // console.log(response);
+  }
+  return (
+    <div>
+      <Formik onSubmit={handleSubmit}>
+        {({
```
Comment (vladimirsan) [inline pages/register.js]: could we extract this outside of the return for cleaner code?

File: pages/register.js [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+                                  {element.label}
+                                </option>
+                              ))}
+                            </Field>
+                            <button
+                              type="button"
+                              className="btn"
+                              onClick={() => remove(index)}
+                            >
+                              <i className="far fa-times-circle" />
```
Comment (vladimirsan) [inline pages/register.js]: Is this indentation expected?

Comment (joejemmely) [reply pages/register.js]: We use [Prettier](https://prettier.io/) for the code formatting

File: pages/register.js (hunk 1 above)
Comment (joejemmely) [reply pages/register.js]: we will amend this and have use environment variables

File: pages/register.js (hunk 3 above)
Comment (joejemmely) [reply pages/register.js]: Not really, Formik uses the render props pattern (either props.children or render={()=>{...}})

Source : [Formik documentation](https://jaredpalmer.com/formik/docs/api/formik)

**Pull request merged**